packages/schema/src/camelCase.ts

```diff
@@ -41,7 +41,7 @@ export class CamelCase extends PrismaReader {
             }
             if (cleanLine[1].includes('_') && kind !== 'enum') {
```

Comment:
look here in the statement. It will not update the word if not include `_`, so I think you didn't test your code